ldk/javascript/src/webpack/config.ts

```diff
@@ -37,7 +36,25 @@ const config: webpack.Configuration = {
     rules: [
       {
         test: /\.ts$/,
-        use: 'ts-loader',
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              sourceType: 'unambiguous',
+              presets: [
+                [
+                  '@babel/preset-env',
+                  {
+                    useBuiltIns: 'entry',
+                    corejs: '3.11',
+                  },
+                ],
+              ],
+              plugins: ['@babel/plugin-transform-destructuring', '@babel/plugin-transform-runtime'],
```

Comment:
Curious: why was`@babel/plugin-transform-destructuring` plugin selected. I don't see us using this feature anywhere inside of the `/javascript` project. Is it required by a 3rd party npm module?

Reply:
So destructing support lets us do this: https://babeljs.io/docs/en/babel-plugin-transform-destructuring. It's not something we're consuming inside the LDK itself I don't think, but our LDK users will often do that.

There's a further question about how we let consumers add additional plugins.